docs/en/operations/settings/settings.md

```diff
@@ -1258,4 +1258,55 @@ Possible values:
 
 Default value: 16.
 
+## low_cardinality_max_dictionary_size {#low_cardinality_max_dictionary_size}
+
+Sets a maximum size in rows of a shared global dictionary for the [LowCardinality](../../sql-reference/data-types/lowcardinality.md) data type that can be written to a storage file system. This setting prevents problems with RAM in case of unlimited dictionary growth. All the data that was not dictionary encoded are written in not encoded way as ClickHouse do for non-`LowCardinality`-columns.
+
+Possible values:
+
+-   Any positive integer.
+
+Default value: 8192.
+
+## low_cardinality_use_single_dictionary_for_part {#low_cardinality_use_single_dictionary_for_part}
+
+Turns on or turns off using of single dictionary for data part.
+
+By default, ClickHouse server monitors size of dictionaries and if a dictionary overflows then server starts to write the next one. To prohibit creating several dictionaries set `low_cardinality_use_single_dictionary_for_part = 1`.
+
+Possible values:
+
+- 1 — Creating of several dictionaries for data part is prohibited.
+- 0 — Creating of several dictionaries for data part is not prohibited.
+
+Default value: 0.
+
+## low_cardinality_allow_in_native_format {#low_cardinality_allow_in_native_format}
+
+Allows or restricts using the [LowCardinality](../../sql-reference/data-types/lowcardinality.md) data type with the [Native](../../interfaces/formats.mdnative) format.
+
+If usage of `LowCardinality` is restricted, ClickHouse server converts `LowCardinality`-columns to ordinary for `SELECT` queries, and convert ordinary columns to `LowCardinality`-columns for `INSERT` queries.
+
+Possible values:
+
+- 1 — Usage of `LowCardinality` is not restricted.
+- 0 — Usage of `LowCardinality` is restricted.
+
+Default value: 1.
+
+
+## allow_suspicious_low_cardinality_types {#allow_suspicious_low_cardinality_types}
+
+Allows or restricts using [LowCardinality](../../sql-reference/data-types/lowcardinality.md) with data types of small fixed size (8 bits or less).
+
+When allowed, merge times and memory consumption can increase.
```

Comment:
```suggestion
When allowed, merge times and memory consumption can increase.
```
* probably needs a reference to MergeTree 
* not clear: memory consumption of what?

Reply:
For small fixed values `LowCardinality` is usually useless because, we always need to store indexes for each row (which are numeric itself). In result:
* Disk space will unlikely be decreased
* Ram consumption may be bigger or smaller. It depends on dictionary size. But it is not very important because usually this consumptions is not significant.
* Some functions may be slower, because often columns are converted from LC back to full.